.kitchen.dokken.yml

```diff
@@ -1,18 +1,22 @@
 ---
 driver:
   name: dokken
-  privileged: true  # because Docker and SystemD/Upstart
+  # because Docker and SystemD/Upstart
+  privileged: true
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
```

Comment:
The old way was the correct way. Or at least the new recommended way. I'd like to roll the provisioner stuff out everywhere eventually 👍 

https://docs.chef.io/config_yml_kitchen.html